src/views/Online/OnlineGameView.jsx

```diff
@@ -12,29 +12,26 @@ import {
   getPlayerStatusProperty,
   playersLobbyStatusLabels,
 } from "../../services/player.service.mjs";
+import {
+  leaveGameOnline,
+  setPlayerStatus,
+} from "../../services/game.online.service.mjs";
 
 export function OnlineGameView({ ...props }) {
   const { gameId } = useParams();
   const { status: gameStatus, data: gameData } = useGame(gameId);
   const [loadGame, setLoadGame] = useState(true);
   const history = useHistory();
-  const [playerIsReady, setPlayerIsReady] = useState(false);
 
   useEffect(() => {
     if (gameStatus === "success") {
       setLoadGame(false);
-      console.log(gameData);
     }
   }, [gameStatus]);
 
-  useEffect(() => {
-    //TODO: set player ready in game ref
-    if (playerIsReady) {
-      console.log("set player ready in gameRef");
-    } else {
-      console.log("set player not ready in gameRef");
-    }
-  }, [playerIsReady]);
+  const playerStatusHandler = async (status) => {
```

Comment:
brauchst du den wrapper hier? SInd beides async funtions und das einzige, was sich ändert, ist, dass der return der inneren funktion nicht weitergegeben wird, oder?

Reply:
Ja stimmt. Mir fiel nur gerade ein das ein try and catch da gut wäre, da diese nicht innerhalb unserer Funktionen ist. Habs mal im try and catch gepackt, wodurch der wrapper einen Sinn bekommt.